packages/w3/bin.js

```diff
@@ -5,17 +5,24 @@ import w3 from './index.js'
 const cli = meow(
   `Usage
     --get bafybeidd2gyhagleh47qeg77xqndy2qy3yzn4vkxmk775bg2t5lpuy7pcu --output pics --api http://127.0.0.1:8787
+    --put /path/to/files
```

Comment:
Not for this PR but I think it would be nice if these were subcommands not flags...i.e. `w3 get CID` and `w3 put /file`.

The [carbites CLI](https://github.com/nftstorage/carbites-cli/) does this with meow.

Reply:
agree! i want commands too. I was thinking of using a lib other than meow that was more geared towards clis with sub commands. https://github.com/lukeed/sade looks nice, as used by the admin tooling for nft.storage https://github.com/ipfs-shipyard/nft.storage/blob/main/packages/tools/cli.js